code/prime-number/Model/src/main/java/ru/unn/agile/primenumber/model/PrimeNumberFinder.java

```diff
@@ -0,0 +1,61 @@
+package ru.unn.agile.primenumber.model;
+
+public class PrimeNumberFinder {
+
+    private int startSpan;
+    private int endSpan;
+
+    public PrimeNumberFinder(final int startSpan, final int endSpan) {
+        if (startSpan < 0 || endSpan < 0) {
+            throw new IllegalArgumentException(
+                    "Span's borders should be positive!");
+        }
+        if (startSpan > endSpan) {
+            throw new IllegalArgumentException(
+                    "Span's right border should be bigger than left border!");
+        }
+        this.endSpan = endSpan;
+        this.startSpan = startSpan;
+    }
+
+    public String findNumbers() {
```

Comment:
Why do you use String as result?
Why not List<int> ?

Reply:
Because as I know, in second lab it should be printed in UI, and it should be beautiful and user-friendly.
Why should I do ArrayList -> Beautiful Formatter -> String 
if i can just do beautiful string as return value.

Compare:
1 2 3 4 5
and
[1, 2, 3, 4, 5]

What is prettier? 
It is obvious.

Reply:
But you have to do 1st lab. And I can't find any reason to use String here